site/gatsby-site/src/pages/account.js

```diff
@@ -15,7 +16,7 @@ const Account = (props) => {
 
   return (
     <Layout {...props}>
-      <AiidHelmet>
+      <AiidHelmet canonicalUrl={config.gatsby.siteUrl + '/account'}>
```

Comment:
It looks like  `config.gatsby.siteUrl` could be inside `AiidHelmet` 🤔

Reply:
Good point. Fixed!